README.md

```diff
@@ -429,12 +429,24 @@ stelace.auth.logout()
 To detect when the user session has expired and handle it properly, you can listen to the following error event:
 ```
 stelace.onError('userSessionExpired', function () {
-  // remove authentication tokens via the token store
-  // notify users that they must reconnect
-  // ...
+  // Remove authentication tokens via the token store, e.g.
+  const tokenStore = stelace.getTokenStore()
+  tokenStore.removeTokens()
+
+  // In browser, you can also notify users that they must reconnect, e.g.
+  this.notify('Your session has expired.')
+
+  // And removes any information about the current user, e.g.
+  this.$store.dispatch('logout', { sessionExpired: true })
```

Comment:
No, this is specific to Vue and can be confusing, please just I include what I suggested.